src/objects/Batch_Template__c.object

```diff
@@ -0,0 +1,299 @@
+        <label>Process Using Scheduled Job</label>
+        <trackTrending>false</trackTrending>
+        <type>Checkbox</type>
+    </fields>
+    <fields>
+        <fullName>RequireTotalMatch__c</fullName>
+        <defaultValue>false</defaultValue>
+        <description>When checked, the total number of gifts and total amount must match the expected totals entered for the batch.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>When checked, the total number of gifts and total amount must match the expected totals entered for the batch.</inlineHelpText>
+        <label>Require Expected Totals Match</label>
+        <trackTrending>false</trackTrending>
+        <type>Checkbox</type>
+    </fields>
+    <fields>
+        <fullName>Run_Opportunity_Rollups_while_Processing__c</fullName>
+        <defaultValue>false</defaultValue>
+        <description>When checked, NPSP calculates donor statistics when donations are processed. If unchecked, donor statistics are calculated during the default nightly Scheduled Job. Note that selecting this checkbox may slow down processing of this batch</description>
+        <externalId>false</externalId>
+        <inlineHelpText>When checked, NPSP calculates donor rollups when donations are processed. If unchecked, donor rollups are calculated during the default nightly Scheduled Job. Note that selecting this checkbox may slow down processing of this batch.</inlineHelpText>
+        <label>Calculate Donation Rollups with Batch</label>
+        <trackTrending>false</trackTrending>
+        <type>Checkbox</type>
+    </fields>
+    <label>Batch Template</label>
+    <listViews>
+        <fullName>All</fullName>
+        <columns>NAME</columns>
+        <columns>Description__c</columns>
+        <columns>CREATED_DATE</columns>
+        <columns>CREATEDBY_USER</columns>
+        <filterScope>Everything</filterScope>
+        <label>All</label>
+    </listViews>
+    <nameField>
+        <label>Template Name</label>
+        <type>Text</type>
+    </nameField>
+    <pluralLabel>Batch Templates</pluralLabel>
+    <searchLayouts>
+        <searchResultsAdditionalFields>Description__c</searchResultsAdditionalFields>
+        <searchResultsAdditionalFields>CREATED_DATE</searchResultsAdditionalFields>
+        <searchResultsAdditionalFields>CREATEDBY_USER</searchResultsAdditionalFields>
+    </searchLayouts>
+    <sharingModel>ReadWrite</sharingModel>
+    <visibility>Public</visibility>
+    <webLinks>
+        <fullName>Edit_Processing_Settings</fullName>
+        <availability>online</availability>
+        <displayType>button</displayType>
+        <encodingKey>UTF-8</encodingKey>
+        <height>600</height>
+        <linkType>url</linkType>
+        <masterLabel>Edit Processing Settings</masterLabel>
+        <openType>sidebar</openType>
+        <protected>false</protected>
+        <url>/apex/BGE_BatchTemplateOverride?Id={!Batch_Template__c.Id}</url>
```

Comment:
This won't work in an un-namespaced org when NPSP is managed. Do this instead:

```suggestion
        <url>/apex/{!$Setup.Package_Settings__c.Namespace_Prefix__c}BGE_BatchTemplateOverride?Id={!Batch_Template__c.Id}</url>
```

Reply:
Fixed this in the namespace consistency PR.